external-docs/docs/guides/errors/build-command-errors/attribute-not-linked.md

```diff
@@ -0,0 +1,11 @@
+# Error - Attribute Not Linked
+
+## When it occurs
+
+A units or attribute column is defined but it is not linked to any observation value column.
+
+## How to fix
+
+In the pivoted shape a unit or attribute column must be directly linked to a observation column. 
```

Comment:
Could you link the user through to the `shaping your data` page? It'll provide some additional context to figure out what they did wrong.